reactioncommerce/reaction
# Order managers should be able to see a risk evaluation for an order

Resolves #2322. 

**How To Test:**
* Using Stripe as payment gateway, make an order. Use the card number `4000000000009235`. This is a test card from Stripe that succeeds with an elevated risk response. See comment below.
* Check the Order Dashboard for that order, you should see it flagged as elevated.
* Switch to Example Payment Method from Stripe, and make another order using the same card above. With this, the Example Payment flags the charge as high risk.
* Check the Order Dashboard for that order, you should see it flagged as high risk (red).
* Also confirm that there is a warning alert before capture of an elevated or high risk order in the Order summary side panel.
* Confirm that making orders with other normal cards (e.g 424242424242...) works as before without any side effect from this PR

NB: (i) The other included payment methods can be updated later based on how each processes risk evaluation. I updated only Stripe package & Example Method in this PR.
(ii) Bulk capturing doesn't popup an alert on risky orders capturing at the moment. That'll be handled in #2762 

**More Details about this PR:**
- Adds a risk level field on the paymentMethod schema (marked as optional). It will allow one of three values `normal`, `elevated` or `high`. So each payment package (when we come around to update them) will only need to set this field based on how they process risk levels. The example used for Stripe & Example payment here can be followed.
- Based on the value of this filed we display badges on the UI.
- We also show an alert when capturing an order that's flagged as risky.

Link
* [Stripe Risk Evaluation Docs](https://stripe.com/docs/radar/risk-evaluation)
